sql/moz-fx-data-shared-prod/fenix/attributable_clients_v2/view.sql

```diff
@@ -0,0 +1,36 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod`.fenix.attributable_clients_v2
```

Comment:
I thought views should not include version numbers?

Reply:
This is a special view; it adds some fields via joins that I need downstream.

On the other hand, we're not quite ready to replace v1, due to some _different_ downstream dependencies. As such, I'm putting the logic in a v2 view, and when we're ready, we'll point the overall view `fenix.attributable_clients` at this view instead.